modules/rollback-operator-upgrades.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-operator.adoc
+:_module-type: PROCEDURE
+[id="rollback-operator-upgrades_{context}"]
+= Rolling back Operator upgrades
+
+[role="_abstract"]
+While upgrading to a new {rh-rhacscs-first} version, if the new upgrade results in errors or other unexpected issues during the upgrade, you can roll back to the previously installed version of {product-title-short}.
+
+[IMPORTANT]
+====
+//Rewrite TBD
+If the operator does manage to upgrade a central instance during an intermediate upgrade, then the operator will need to be downgraded to that version in order to use the db backup
+
+- alternatively the central deployment is removed from operator control and manually updated to the desired image
+- Image specs can be found here
+- For some reason, OLM will upgrade to the first version of a minor release when performing intermediate upgrades.
+
+Example scenario: 3.71.3 -> 3.72.0 -> 3.73.0 -> 3.73.2
+
+In this scenario, the ACS operator performed a Central upgrade while running on 3.72.0 and on 3.73.2.  If a rollback is attempted, it must be to 3.72.0.  When updating the OLM subscription to the 3.72 channel, OLM will deploy the 3.72.3 version of the operator.  There is no value of the forceRollbackVersion config parameter that will successfully trigger a rollback:
+- 3.72.0 won’t match the version of Central deployed, so it will not enable DB rollback
+- 3.72.3 will enable DB rollback, but it does not match the version of the DB backup
+====
+
+
+.Prerequisites
+//Any prerequisites?
+
+.Procedure
+. Delete CSV and {product-title-short} OLM subscription:
++
+[source,terminal]
+----
+codeoc -n rhacs-operator delete csv rhacs-operator.v4.0.0
+----
+//Find out the complete process for rolling back from 4.0 to 3.74
++
+[source,terminal]
+----
+oc -n rhacs-operator delete subscription rhacs-operator
+----
+. Create a new subscription with the update channel:
++
+[source,terminal]
+----
+$ cat << EOF | oc -n rhacs-operator apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: rhacs-operator
+spec:
+  channel: rhacs-3.73
+  name: rhacs-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Automatic
+EOF
+----
+//TBD
+
+//.Verification
+//Provide the user with verification methods for the procedure, such as expected output or commands that confirm success or failure.
```

Comment:
Central is up and running, they can log into Central and use the product

For 4.0 -> 3.74 it should also remove the central-db deployment?